### ISSUE TITLE
Suddenly no longer working on 10.11.4

### ISSUE THREAD

**Issue by colindean** [JohnCoates/Aerial#239]
I'm getting the dreaded

> You cannot use the Aerial screen saver with this version of OS X.
> Please contact the vendor to get a newer version of the screen saver.

I'm not sure what changed; this was literally working this morning and I've not done any OS updates...

**Comment by colindean** [comment]
Nevermind! I thought I was up to date, but I'm not.

**Comment by alistairmcmillan** [comment]
Hi @colindean if you're curious, here is why you got that message. It's a bug in OS X.

http://alistairmcmillan.github.io/2016/04/25/OSX-Swift-screensaver-bug.html

**Comment by colindean** [comment]
Interestingly enough, I hit this very same issue while messing around with starting another screensaver of my own. I wrote my own [Bitcoin Price Screensaver](https://github.com/colindean/BitcoinPriceScreensaver) in Objective-C in 2013 and wanted to do up something else just this past weekend. I don’t think I came across that same article, but another article said something like “if you get the dreaded ‘ask vendor for a new version’ message, it means you forgot to do <step that embeds swift differently>”. Thanks for the article, though, it was a good read.
